eng/SourceBuild.props

```diff
@@ -84,6 +84,7 @@
   </Target>
 
   <Target Name="RestoreNpmPackages"
+          Condition="'$(BuildNodeJS)' == 'true'"
```

Comment:
In which scenario is valid to skip building nodejs. The Component package won't build and the build will fail.